masterlist.yaml

```diff
@@ -7375,3 +7375,11 @@ plugins:
       - link: 'https://www.nexusmods.com/fallout4/mods/66724/'
         name: 'Diamond City Expansion - eXoPatch'
     after: [ 'PRP.esp' ]
+
+  - name: 'Diello_ExtraIconsForFIS.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/68889/'
+        name: 'Extra Icons for FIS'
+    after:
+      - 'FIS-Naming-Weap-Armo-EN.esp'
+      - 'M8r Complex Sorter.esp'
```

Comment:
The load after for `M8r Complex Sorter.esp` needs to be removed, see [this comment on Discord](https://discord.com/channels/830436661736243230/915413649638707251/1142802194777772162) from `m8r98a4f2`, author of the Complex Sorter.